tests/ops/models/test_custom_report.py

```diff
@@ -69,15 +86,24 @@ def test_update_custom_report(self, db: Session, user: FidesUser, custom_report)
             db=db,
             data={
                 "config": CustomReportConfig(
-                    column_map={"system": "System"}
+                    column_map={
+                        "system": {
+                            "label": "System",
+                        },
+                    }
                 ).model_dump(mode="json"),
             },
         )
         assert updated_report.name == "Custom report"
         assert updated_report.type == ReportType.datamap
         assert updated_report.created_by == user.id
         assert updated_report.config == {
-            "column_map": {"system": "System"},
+            "column_map": {
+                "system": {
+                    "enabled": None,
```

Comment:
We should define a default behavior if `enabled` isn't provided. This follows the principle of "Be liberal in what you accept, and conservative in what you send."

In this case we should be able to accept True, False, or None, but we should be strict and only return True or False. This way, anyone consuming our API doesn't need to handle the scenario where `enabled` is None, they can always expect a boolean and not have to deal with undefined behavior.

Reply:
Make sure you add a test that covers whatever you decide for the default behavior